lxd/instance/drivers/driver_qemu.go

```diff
@@ -3727,7 +3727,11 @@ func (d *qemu) addRootDriveConfig(qemuDev map[string]string, mountInfo *storageP
 				clusterName = storageDrivers.CephDefaultUser
 			}
 
-			driveConf.DevPath = device.DiskGetRBDFormat(clusterName, userName, config["ceph.osd.pool_name"], vol.Name())
+			// Driver and pool name arguments can be ignored as CephGetRBDImageName doesn't need them
+			vol := storageDrivers.NewVolume(nil, "", storageDrivers.VolumeTypeVM, storageDrivers.ContentTypeBlock, vol.Name(), nil, nil)
```

Comment:
why are we creating a new locally scope `vol` here rather than using the one above from line 3715?

Reply:
I'm thinking that it would be clearer to update `CephGetRBDImageName` to accept specific arguments that it needs rather than a `Volume` type, as it seems several places are having to craft specific `Volume` vars to make it work.

Reply:
I missed the `vol` on line 3715, thanks.

`CephGetRBDImageName` uses `vol.ConfigBlockFilesystem()`, so we need the volume's config as well. I've updated all usages so that the config is passed as well, rather than using `NewVolume`.